frontend-react/src/pages/details/FacilitiesTable.tsx

```diff
@@ -1,18 +1,81 @@
-import ReportResource from '../../resources/ReportResource';
+import { useOktaAuth } from '@okta/okta-react';
+import { useState, useEffect } from 'react';
+import Spinner from '../../components/Spinner';
+import AuthResource from '../../resources/AuthResource';
+import { groupToOrg } from '../../webreceiver-utils';
 
 interface Props {
     /* REQUIRED
     Passing in a report allows this component to map through the facilities property
     to display a row per facility on the FaclitiesTable. */
-    report: ReportResource | undefined
+    reportId: string | undefined
 }
 
+/* INFO
+   This type exists as a part of the tempoarary fix and can be removed after we switch
+   back to using useResource() for that call
+   >>> Kevin Haube, Sept 27, 2021 */
+type Facility = {
+    organization: string | undefined
+    facility: string | undefined
+    location: string | undefined
+    CLIA: string | undefined
+    positive: string | undefined
+    total: string | undefined
+}
+
+
 function FacilitiesTable(props: Props) {
-    const { report } = props
+    const { reportId } = props;
+
+    /* DEBUG
+       This will be our approach to getting facilities from the API once rest-hooks is 
+       configured properly
+       >>> Kevin Haube, Sep 24, 2021 */
+
+    // const facilities: FacilityResource[] = useResource(FacilityResource.getFacilities(reportId), {})
+
+    /* INFO
+       This is a temporary fix while I work on learning how to configure custom endpoints
+       and calls with the rest-hooks library. 
+       >>> Kevin Haube, Sep 24, 2021 */
+
+    const [facilities, setFacilicites] = useState<Facility[]>([]);
+    const { authState } = useOktaAuth();
+
+    useEffect(() => {
+        getFacilities()
+    }, [])
+
+    const getFacilities = async () => {
+        const organization = groupToOrg(
+            authState!.accessToken?.claims.organization.find(o => !o.toLowerCase().includes('sender'))
+        );
+        const headers = new Headers({
+            'Authorization': `Bearer ${authState?.accessToken?.accessToken}`,
+            'Organization': organization!
+        });
+
+        const response = await fetch(`${AuthResource.getBaseUrl()}/api/history/report/${reportId}/facilities`, {
+            method: 'GET',
+            headers: headers
+        })
+
+        const data = await response.json()
+        setFacilicites(data)
```

Comment:
Ideally we have some network and server error handling. Probably need to think through this more holistically since it's not just a problem here

Reply:
@sean-usds I've added #2368 to the Work To-Do section of the PR description to account for this; good call!